controllers/flashsystemcluster_controller_test.go

```diff
@@ -41,6 +41,7 @@ var _ = Describe("FlashSystemClusterReconciler", func() {
 		FlashSystemName  = "flashsystemcluster-sample"
 		namespace        = "openshift-storage"
 		secretName       = "fs-secret-sample"
+		podName          = "ibm-storage-odf-operator"
```

Comment:
could you change `podName` to `operatorPodName`?

Reply:
changed